### PR TITLE
Create instance of BitByte

### PR DESCRIPTION
**Pull request by sam4215**
Perfectly legal under new Java.

Creates an instance of BitByte's sponge forums account.

### CODE REVIEW

**Comment by sam4215** [comment]
Added the error handler @12AwsomeMan34

**Comment by BitByteMC** [comment]
Top kek

File: Error.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+public class Error {
+  blameItOnBitByte.exe();
+  printStackTrace();
+  Waffle waffle = new ButterDev.waffle();
+  sam4215 == ButterDev;
+  out.println("BitByte.exe has stopped working. " + joke.toString());
```
Comment (simon816) [inline Error.java]: lol you broke github syntax highlighting

Comment (sam4215) [inline Error.java]: Aww sweet

How come GitHub no support Java 6526??

Comment (BitByteMC) [inline Error.java]: Because if else false returns true. :neckbeard:

**Pull request merged**